simulation/halsim_ds_socket/src/main/native/cpp/main.cpp

```diff
@@ -124,33 +124,42 @@ static void SetupUdp(wpi::uv::Loop& loop) {
     });
   });
   simLoopTimer->Start(Timer::Time{100}, Timer::Time{100});
+  // DS Timeout
+  int timeoutMs = 100;
+  auto envTimeout = std::getenv("DS_TIMEOUT_MS");
+  if (envTimeout) {
+    timeoutMs = std::stoi(envTimeout);
```

Comment:
Probably should catch the exceptions stoi throws and report them (leaving timeoutMs at default)?

Reply:
Report them to where? I don't think DS error reporting works in sim. Standard output? Standard error?